Week2/num5.js

```diff
@@ -0,0 +1,18 @@
+// 5
+console.log(Math.round(7.25));
+// 5.1
+let z = 7.25 ;
```

Comment:
You added a 'space' here, which is not necessary.

So you can just write:

`let z = 7.25;`